src/test/java/uk/ac/ebi/atlas/search/GeneSearchDaoIT.java

```diff
@@ -103,25 +110,26 @@ void invalidGeneIdReturnsEmpty(String geneId) {
     }
 
     @ParameterizedTest
-    @CsvSource({"'ENSMUSG00000005360', 'E-GEOD-99058', 6"})
+    @CsvSource({"'ENSG00000130066', 'E-GEOD-81547', 29"})
     void validExperimentAccessionReturnsClusterIDsWithPreferredKAndMinP(String geneId,
                                                                         String experimentAccession,
                                                                         Integer preferredK) {
         var minimumMarkerProbability = subject.fetchMinimumMarkerProbability(experimentAccession).get(geneId);
         var result =
                 subject.fetchClusterIdsWithPreferredKAndMinPForExperimentAccession(
                         geneId, experimentAccession, preferredK, minimumMarkerProbability);
+        System.out.println("result:"+result);
 
         assertThat(result)
                 .isNotEmpty()
                 .containsAllEntriesOf(
                         ImmutableMap.of(
-                                6, ImmutableList.of(3),
-                                4, ImmutableList.of(2)));
+                                29, ImmutableList.of(28)));
+        // May be we need to improve this test a bit, But I am fixing this one temporary
```

Comment:
What in this test would need to be improved? Can you please add a bit of more explaination?